Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM perl:latest
+
+ENV PATH="/root/bystro/bin:/usr/local/go/bin:/root/go/bin/:${PATH}"
+ENV PERL5LIB="/root/perl5/lib/perl5:/root/bystro/lib:${PERL5LIB}"
+ENV GOPATH="/root/go"
+
+RUN cpanm --local-lib=/root/perl5 local::lib && eval $(perl -I /root/perl5/lib -Mlocal::lib)
+ADD ./ /root/bystro/
+RUN apt-get update && apt-get install sudo
+
+WORKDIR /root/bystro
+RUN . install/install-lmdb-linux.sh
+RUN wget https://dl.google.com/go/go1.11.linux-amd64.tar.gz \
+    && tar -xf go1.11.linux-amd64.tar.gz \
+    && mv go /usr/local
+
+RUN . install/install-go-packages.sh
+RUN . install/install-perl-libs.sh
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Not following: File not included in mock.](https://app.codacy.com/manual/akotlar/bystro/pullRequest?prid=4887428)